src/client/pages/AdminsPage/index.tsx

```diff
@@ -43,21 +61,33 @@ class AdminsPage extends React.Component<Props, AdminsPageState> {
   componentDidMount() {
     this.props.showLoading();
 
+    // Hide loading state after the API returns the admins
     this.loadAdmins()
       .then(() => this.props.hideLoading());
   }
 
+  /**
+   * Toggle the isRegisterModalOpen state variable to show or hide new admin modal
+   */
   toggleRegisterModal = () => this.setState({
     isRegisterModalOpen: !this.state.isRegisterModalOpen,
   });
 
+  /**
+   * Create a new admin in the database and update the frontend to reflect the change
+   * @param {NewAdminModalFormData} newAdmin the new admin to be added to the system
```

Comment:
Should there be some kind of separator (e.g. `-` or tab or 2 spaces) between the param name and the description? Think that'll make it easier to read. Also capitalize first char of sentence (or not) for consistency.